api/src/main/resources/workbench.yaml

```diff
@@ -3634,14 +3661,44 @@ definitions:
         format: int64
         description: Milliseconds since the UNIX epoch.
 
-  ClusterListResponse:
+  DefaultClusterResponse:
     type: object
     required:
       - defaultCluster
     properties:
       defaultCluster:
         $ref: "#/definitions/Cluster"
 
+  ListClusterResponse:
```

Comment:
Is there a need to return data about the clusters that were just paused? I'd be inclined to call YAGNI (you ain't gonna need this) on this piece. We can easily add more return info if required, but I don't seen an obvious use case for the client to use this data.

Reply:
I am fairly confident that this is useful data, because if one cluster didn't delete for some reason, we can display information about that on the client side by filtering on cluster status on the returned values. In general, I think of deleting clusters as an 'update to be deleted asynchronously', and believe that update requests should return the new values